Makefile

```diff
@@ -171,7 +171,8 @@ code-cov-report: $(CODE_COVERAGE_FILE_TXT)
 # Test E2E
 test-e2e:
 	$(eval export E2E="ok")
-	go test ./test/e2e/... -v
+	go test ./test/e2e/mutate -v
```

Comment:
Only change here is splitting this into two `go test` runs as @realshuting figured out (parallel test runs introduce issues)

Reply:
Thinking more about running tests in sequence, it could slow down the tests as we add more in the future. I think the best approach is to define test policies with limited scope, for example, restrict the policy with a specific target resource, namespace, etc.  Besides, running different types of tests parallelly could help test the resiliency of Kyverno (this may beyond the goal of e2e tests, it's more of chaos testing, cc @mhmohona ). This would of course require extra efforts when adding new tests (as we don't want to introduce any conflicts).

For now, we can schedule tests sequentially, just keep in mind this could be one way of speeding up tests.